packages/base-map/src/styled.ts

```diff
@@ -1,5 +1,17 @@
+import { Popup as MapGlPopup } from "react-map-gl";
 import styled from "styled-components";
 
+/**
+ * Adds a box shadow and tweaks border radius to make popups easier to read.
+ */
+export const Popup = styled(MapGlPopup)`
+  & > .maplibregl-popup-content,
+  & > .mapboxgl-popup-content {
+    border-radius: 6px;
```

Comment:
```suggestion
    border-radius: 10px;
```
I think if we bump this up we should commit to a more round look

Reply:
Updated in https://github.com/opentripplanner/otp-ui/pull/471/commits/45ca046b59d4c46d94a43292084bf17a1e50cec0.